.pre-commit-config.yaml

```
@@ -36,3 +36,8 @@ repos:
     hooks:
       - id: black
         exclude: ^tests/codegen/snapshots/python/
+
+  - repo: https://github.com/asottile/blacken-docs
```

```suggestion
  - repo: https://github.com/adamchainz/blacken-docs
```

just noticed the docs have the wrong url, I'll update it for you!